tsconfig.json

```diff
@@ -0,0 +1,22 @@
+{
+  "compilerOptions": {
+    "allowJs": true,
+    "baseUrl": "./",
+    "esModuleInterop": true,
+    "jsx": "react-jsx",
+    "module": "es6",
+    "moduleResolution": "bundler",
+    "noImplicitAny": true,
+    "noUncheckedIndexedAccess": true,
+    "outDir": "./dist/",
+    "paths": {
+      "@/*": [ "./modules/*" ]
+    },
+    "resolveJsonModule": true,
+    "skipLibCheck": true,
+    "strict": true,
+    "target": "es5",
```

Comment:
Does this match some support target? In all likelihood you can go much higher here, ES5 / ES2009 has been universally supported for more than a decade.

Reply:
This would be a copy paste from gutenberg/remote-data-blocks that I didn't alter until we actually migrated a JS file to TS. I'd agree with you in that, we can definitely go a lot higher but just left that for a follow up migration PR.